src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1586,6 +1586,8 @@ class ApplicationManagerImpl
 
   uint32_t apps_size_;
 
+  bool is_registered_in_timeout_;
```

Comment:
@ydementieiev suggest to use atomic bool as timer might be in a data race with another thread

Reply:
@AKalinich-Luxoft done in [fe7b393](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/fe7b393fe80f89ac9137d6dd9b41a0dfd6ba28c6)